app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :posts, foreign_key: :author_id
+  has_many :comments
+  has_many :likes
```

Comment:
- [ ]  Good job setting the associations between models bettween the User model and other models 👏 , I will suggest you to add the `foreign_key: :author_id` as following 💯 . It will establish the one-to-many relation bettween models as describe [here in the erb diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png).

```ruby
has_many :posts, foreign_key: :author_id
has_many :comments, foreign_key: :author_id
has_many :likes, foreign_key: :author_id
```